pkgs/servers/news/leafnode1/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchurl, pcre2, libxcrypt }:
+
+let
+  pname = "leafnode";
+  version = "1.12.0";
+in stdenv.mkDerivation {
```

Instead of let-against-rec, use

```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "leafnode";
  version = "1.12.0";
```